src/templates/blog-single.js

```diff
@@ -46,7 +46,7 @@ const BlogSinglePage = ({data}) => {
     <ThemeProvider theme={theme}>
       <Layout>
         <GlobalStyle />
-        <SEO title={data.mdx.frontmatter.title} />
+        <SEO title={data.mdx.frontmatter.title} image={data.mdx.frontmatter.thumbnail.publicURL}/>
```

Comment:
I remember @Jashpatel1 telling me that this doesn't work :thinking:

Reply:
I tried it using `gatsby build` on my local and the image was coming up in the meta tags, while the current site has the `Layer5` logo.

Reply:
Strange